working-examples/reflow-carousel-panel-horizontal-scroll/index.html

```diff
@@ -0,0 +1,306 @@
+                        } else {
+                            console.log(`scroll right by: ${cardWidth}`);
+                            carousel.scrollBy({
+                                left: cardWidth,
+                                behavior: 'smooth'
+                            });
+                        }
+                    };
+
+                    prevButton.addEventListener('click', scrollLeft);
+                    nextButton.addEventListener('click', scrollRight);
+                    carousel.addEventListener('scroll', updateButtons);
+
+                    updateButtons();
+
+                    document.addEventListener('keydown', (e) => {
+                        if (e.key === 'ArrowLeft') {
+                            scrollLeft();
+                        } else if (e.key === 'ArrowRight') {
+                            scrollRight();
+                        }
+                    });
+                })();
+            });
+        </script>
+    </head>
+    <body>
+        <div class="carousel">
+            <div>
+                <h2 id="news">New Jackets</h2>
+                <div class="prod-list" role="region" tabindex="0" aria-labelledby="news">
+                    <ul>
+                        <li>
+                            <div class="img-box">
+                                <img src="jackets.png" alt="">
+                            </div>
+                            <a class="prod-name" href="#">Giacomo's Jacket</a>
+                            <p>A classic biker-style jacket with quilted shoulders and a belted waist</p>
+                            <p class="price">$500</p>
+                            <span>Colors</span>
+                            <ul class="colors">
+                                <li class="black" title="black">Black</li>
+                                <li class="brown" title="brown">Brown</li>
+                            </ul>
+                        </li>
+                        <li>                            
+                            <div class="img-box">
+                                <img src="jackets.png" alt="" style="transform: translate(-12.5%, 0%)">
+                            </div>
+                            <a class="prod-name" href="#">Scott's Jacket</a>
+                            <p>A vibrant, colorful bomber jacket with a unique, almost flame-like pattern</p>
+                            <p class="price">$500</p>
+                            <span>Colors</span>
+                            <ul class="colors">
+                                <li class="black" title="black">Black</li>
+                                <li class="flame" title="flame">Flame</li>
+                            </ul>
+                        </li>
+                        <li>
+                            <div class="img-box">
+                                <img src="jackets.png" alt="image alterantive" style="transform: translate(-25%, 0%)">
```

Comment:
this alt has 'image alternative' instead of alt="".  guessing this should be removed @giacomo-petri ?

Reply:
thanks, placeholder text that should be removed; replacing with decorative image alt=""